### PR TITLE
JSF 2.2 PartialResponse remove wrapper

### PR DESCRIPTION
**Pull request by melloware**
cc @tandraschko

### CODE REVIEW

File: primefaces/src/main/java/org/primefaces/context/PartialResponseWriterWrapper.java [hunk 1]
```diff
@@ -23,96 +23,13 @@
...
-import java.io.IOException;
-import java.util.Map;
 import javax.faces.context.PartialResponseWriter;
 
 public class PartialResponseWriterWrapper extends PartialResponseWriter {
 
-    private PartialResponseWriter wrapped;
 
     public PartialResponseWriterWrapper(PartialResponseWriter wrapped) {
         super(wrapped);
```
Comment (melloware) [inline primefaces/src/main/java/org/primefaces/context/PartialResponseWriterWrapper.java]: Not even sure we need this class at all anymore?

Comment (tandraschko) [reply primefaces/src/main/java/org/primefaces/context/PartialResponseWriterWrapper.java]: we surely need this
the JSF specs only contains a ResponseWriterWrapper, not a PartialResponseWriterWrapper

Comment (melloware) [reply primefaces/src/main/java/org/primefaces/context/PartialResponseWriterWrapper.java]: Sorry I mean this is wrapping just the wrapper and then being extended when we can just extend PartialResponseWriter directly now?

**Comment by melloware** [comment]
Maybe I don't understand but this wrapper was just wrapping and then calling each method overridden of PartialResponseWriter?  So our Impl wasn't doing anything was it?

**Comment by tandraschko** [comment]
hmm i see
then you need to debug, to find out what is different in Mojarra and MF wrapping the writer

**Comment by melloware** [comment]
@christophs78 may need some help with this one.  NONE of these test fail locally for me.  Very strange.

**Comment by christophs78** [comment]
After checking out your feature-branch and some clean + install (PrimeFaces "Core") respective clean + package (PrimeFaces IT) later it also breaks on my machine with MyFaces 2.3 AND CSP enabled.
image

**Comment by melloware** [comment]
Ahh CSP I think I was not enabled. Let me try that !

**Comment by melloware** [comment]
OK was able to debug it and put my findings in the MyFaces ticket: https://issues.apache.org/jira/projects/MYFACES/issues/MYFACES-4536

**Comment by tandraschko** [comment]
@melloware should we close this?
we have a reproducer in the MF issue and can't remove this for this, maybe in the next 2-3 major PF versions

**Comment by melloware** [comment]
Yep we can close

**Pull request closed without merging**